### PR TITLE
Update whats-new-in-v72.rst

### PR DESCRIPTION
**Pull request by wuwinson**
Updated the summary to reflect the delayed release schedule.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@wuwinson - Would you be open to applying this update directly to the [v7.2-documentation branch](https://github.com/mattermost/docs/pull/5819)? Doing so ensures that we can release all v7.2 functionality at once without having to manage individual PRs for the release.

**Comment by cwarnermm** [comment]
@wuwinson - Given that this page is already live in production, please disregard my request to add this to the v7.2-documentation branch. :)

File: source/welcome/whats-new-in-v72.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 What’s new in Mattermost Boards v7.2
```
Comment (amyblais) [inline source/welcome/whats-new-in-v72.rst:1]: I also noticed that the TOC link is currently broken here https://docs.mattermost.com/guides/welcome-to-mattermost.html#mattermost-interface-changes.

Comment (justinegeffen) [reply source/welcome/whats-new-in-v72.rst:1]: Thanks Amy! Fixed.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**